addons/auth_signup/data/auth_signup_data.xml

```diff
@@ -195,6 +195,98 @@
             <field name="user_signature" eval="False"/>
         </record>
 
+        <!-- Email template for reminder of unregistered users -->
+        <record id="unregistered_user_reminder" model="mail.template">
+            <field name="name">Auth Signup: Unregistered Users</field>
+            <field name="model_id" ref="base.model_res_users"/>
+            <field name="subject">Reminder for unregistered users</field>
+            <field name="email_from">"${object.company_id.name|safe}" &lt;${object.company_id.email or user.email|safe}&gt;</field>
```

Comment:
Maybe you could use object.company_id.partner_id.email_formatted instead. That way you already have a formatted email instead of doing it manually.